stable/etcd-operator/templates/cluster.yaml

```diff
@@ -1,6 +1,6 @@
-{{- if .Values.cluster.enabled -}}
-apiVersion: "coreos.com/v1"
-kind: "EtcdCluster"
+{{- if and .Values.cluster.enabled (not .Release.IsInstall) -}}
```

Comment:
please consider increasing the major version as apiVersion and kind of this TPR is changed, which may break existing deployment. according to semver guidelines. here is a link to it http://semver.org